File: 0001.rb
def solve(min, max)
  sum = 0
  (min...max).select{|i| i % 3 == 0 or i % 5 == 0}.each{|i| sum += i}
  return sum
end

if __FILE__ == $0
  puts "Answer: " + solve(0, 1000).to_s
end

File: 0001.lua
function solve(min, max)
	sum = 0
	for i = min,max-1 do
		if i % 3 == 0 or i % 5 == 0 then
			sum = sum + i
		end
	end
	return sum
end

print("Answer:", solve(0, 1000))

